src/src/AdminBundle/Entity/EventLanguages.php

```diff
@@ -31,15 +31,47 @@ public function getId()
         return $this->id;
     }
 
+    /**
+     * @return mixed
+     */
+    public function getEventId()
+    {
+        return $this->eventId;
+    }
+
+    /**
+     * @param mixed $eventId
+     */
+    public function setEventId($eventId)
+    {
+        $this->eventId = $eventId;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getLanguageId()
+    {
+        return $this->languageId;
+    }
+
+    /**
+     * @param mixed $languageId
+     */
+    public function setLanguageId($languageId)
+    {
+        $this->languageId = $languageId;
+    }
+
     /**
      * @ORM\ManyToOne(targetEntity="Event", inversedBy="languages")
      * @ORM\JoinColumn(name="event_id", referencedColumnName="id")
      */
-    private $event_id;
+    private $eventId;
```

Comment:
toto je `event` a nie `eventId`. Premenuj to vsade.

Reply:
Oki. Premenovala som to vsade v mojom PR. Treba dat pozor aj na vikine PRs.